Update Manual Steps in Getting Started guide (#35898)
vercel/next.js: docs/getting-started.md

- `dev` - Runs [`next dev`](/docs/api-reference/cli.md#development) to start Next.js in development mode
- `build` - Runs [`next build`](/docs/api-reference/cli.md#build) to build the application for production usage
- `start` - Runs [`next start`](/docs/api-reference/cli.md#production) to start a Next.js production server
- `lint` - Runs [`next lint`](/docs/api-reference/cli.md#lint) to set up Next.js' built-in ESLint configuration

Create two directories `pages` and `public` at the root of your application:

- `pages` - Associated with a route based on their file name. For example `pages/about.js` is mapped to `/about`
- `public` - Stores static assets such as images, fonts, etc. Files inside `public` directory can then be referenced by your code starting from the base URL (`/`).

Next.js is built around the concept of [pages](/docs/basic-features/pages.md). A page is a [React Component](https://reactjs.org/docs/components-and-props.html) exported from a `.js`, `.jsx`, `.ts`, or `.tsx` file in the `pages` directory. You can even add [dynamic route](/docs/routing/dynamic-routes) parameters with the filename.

> Inside the `pages` directory add the following file to get started:

Reviewer: ```suggestion
Create a `pages` directory containing a `index.js` file to get started:
```

Lets keep "Create" since you need to create both a directory and the file.

Author: We are already creating `pages` and `public` directories. Please see the line `80`.

> - `index.js` - This is the page that is rendered when the user visits the root of your application

Reviewer: This bullet point feels out of place since normally bullet points have more than one

Author: My bad, I've fixed this.
